cnd/src/network.rs

```diff
@@ -215,33 +218,46 @@ fn derive_key_pair(seed: &RootSeed) -> Keypair {
 #[derive(NetworkBehaviour)]
 #[allow(missing_debug_implementations)]
 pub struct ComitNode {
+    /// The rfc003 network behaviour.
     comit: Comit,
+    /// The new spilt protocols network behaviour.
     comit_ln: ComitLN,
```

Comment:
:question: Would it make sense to call this one `Comit` and the old one `ComitRfc003`?

Reply:
Yes, will add this to my todo list, thanks.